test/FunctionalTests/Tests/DataOData/Tests/OData.Query.TDD.Tests/TestUtilities/HardCodedTestModel.cs

```diff
@@ -894,7 +904,10 @@ internal class HardCodedTestModelXml
         <Key>
           <PropertyRef Name=""ID"" />
         </Key>
-        <Property Name=""ID"" Type=""Edm.Int32"" Nullable=""false"" />        
+
+
+        <Property Name=""ID"" Type=""Edm.Int32"" Nullable=""false"" />
+        <Property Name=""SSN"" Type=""Edm.String"" />

```

Comment:
Yes, for that EdmModel.AddVocabularyAnnotations needs to be invoked. Added changes. Also Dong provided updates for doing this as well.